src/heuristics/labels.ts

```diff
@@ -245,14 +245,14 @@ export function labelsToDisplayFromGrid(params: {
   for (let i = 0, l = worthyLabels.length; i < l; i++) {
     const n1 = worthyLabels[i],
       d1 = cache[n1],
-      p1 = camera.graphToViewport(dimensions, d1.x, d1.y);
+      p1 = camera.graphToViewport(dimensions, d1);
 
     if (collisions.has(n1)) continue;
 
     for (let j = i + 1; j < l; j++) {
       const n2 = worthyLabels[j],
         d2 = cache[n2],
-        p2 = camera.graphToViewport(dimensions, d2.x, d2.y);
+        p2 = camera.graphToViewport(dimensions, d2);
```

Comment:
The reason why all those functions were taking x and y as two arguments was to avoid memory allocation of `{x, y}`. But since we need to create those objects anyway it seems better to do as you changed.